frontend/assets/stylesheets/components/_navigation.scss

```diff
@@ -129,7 +129,7 @@
         .global-navigation__scroll,
         .global-navigation__list,
         .global-navigation__item {
-            display: block;
+            display: block !important;
```

Comment:
I realise that `!important` is being used elsewhere, but it's a fairly dangerous bit of CSS - it would be good to reduce it's usage if possible, are you sure it's needed here? Ideally, we would use BEM principles to avoid overwriting styles like this.

@SiAdcock on the frontend team (sits immediately behind us!) would be happy to discuss alternative ways of fixing this!